source/jobs/sync-jobs.txt

```diff
@@ -1,7 +1,51 @@
+.. _rm-sync-jobs:
+
 =========
 Sync Jobs
 =========
 
+Sync jobs transfer data from your relational database to MongoDB. Sync 
+jobs use project-level :ref:`rm-mapping-rules` to distribute data to 
+your MongoDB database.
+
+Types of Sync Jobs
+------------------
+
+Relational Migrator offers two different sync job types:
+
+**Snapshot**: Snapshot sync jobs run once, typically for a point in time 
```

Comment:
Just trying to give some contrast here in the definition between the term and the definition.